lib/ProductOpener/Tags.pm

```diff
@@ -807,33 +897,73 @@ sub build_tags_taxonomy($$$) {
 	defined $tags_images{$lc} or $tags_images{$lc} = {};
 	defined $tags_images{$lc}{$tagtype} or $tags_images{$lc}{$tagtype} = {};
 
+	# we ofen use the term *tag* in the code to indicate a single entry between commas
+	# that is most lines, are tags separated by commas.
+
+	# when we speak about normalized entry, or tagid,
+	# it's the tag value where we lowercased, replace separators by dash etc.,
+	# see get_string_id_for_lang
+
+	# line tagid (or tagid_c) in particular is the first entry in a line
```

Comment:
I would change both $line_tagid and $tagid_c to $lc_tagid, as the important part is that it's about a specific language.

Reply:
done (next commit)